src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshService.java

```diff
@@ -53,7 +53,8 @@ public MeshService(MeshClient meshClient,
     @Scheduled(fixedRateString = "${labresults.mesh.wakeupIntervalInMilliseconds}")
     public void scanMeshInboxForMessages() {
         if (!meshMailBoxScheduler.isEnabled()) {
-            LOGGER.warn("Not running the MESH mailbox polling cycle because it is disabled. Set variable " +
+            LOGGER.warn("Not running the MESH mailbox polling cycle because it is disabled. Set variable "
+                +
                     "LAB_RESULTS_SCHEDULER_ENABLED to true to enable it.");
```

Comment:
```suggestion
            LOGGER.warn("Not running the MESH mailbox polling cycle because it is disabled. Set variable "
                + "LAB_RESULTS_SCHEDULER_ENABLED to true to enable it.");
```

Reply:
Done in 0d2ce4bcf1